cmds/dhclient/dhclient.go

```diff
@@ -37,6 +37,7 @@ const (
 
 var (
 	leasetimeout = flag.Int("timeout", 15, "Lease timeout in seconds")
+	retry        = flag.Int("retry", -1, "Max number of attempts to send DHCPv6 solicit. -1 means infinity")
```

Comment:
Plus, I am still not quite sure which scenarios require retries, when any errors happen during listening, or only when we cannot listen anything within the time limit. I prefer the former one. 

But if we do the former on DHCPv6, and do retry under networkError.Timeout case in DHCPv4 (which is the latter scenario), I wonder whether there's a non-consensus that misleads.